### ISSUE TITLE
Logid stops working, service restart needed

### ISSUE THREAD

**Issue by MorganMLGman** [PixlOne/logiops#280]
After a reboot logid works for few minutes and then stops working. Service restart is needed with  `sudo systemctl restart logid`. `sudo systemctl status logid` gives no information 
![image](https://user-images.githubusercontent.com/33269270/143298101-37ae62d4-49a4-4dc7-85d0-11eff20e382e.png)

**Comment by tomekand1** [comment]
is that happening after sleep mode activated ??

**Comment by MorganMLGman** [comment]
No, no sleep mode. Just using my keyboard and mouse normalny and after like 10 minutes my gesture button on mouse and function keys on keyboard stops working properly

**Comment by VDuchauffour** [comment]
Same here when my MX Master 3 is deactivated after 10 minutes.

**Comment by GarThor** [comment]
I am also having this issue.  I don't know how long it is before I restart it, and when it stops working though... usually it happens after I've stepped away from my computer for a period of time.

I'm not sure if there are logs somewhere that would help diagnose this issue?  Maybe my mouse goes to sleep, and logid can't detect it when it comes back online?
